### ISSUE TITLE
Warn the user when setting styles through class name on things that should not be changed

### ISSUE THREAD

**Issue by rafaelchiti** [byronmwong/react-flexbox-layout#11]
Would be nice to have the same warnings we have when setting a style with inline styles, for those things that can not be overwritten.
We need this for class names since we are using css modules.
This way we can send a className to HLayout or any of the other components for changing the things that are safe to change but warn the user for those things that are not safe.
We could use computedLayout in `dev` for achieving this.

**Comment by bywo** [comment]
I don't think this should be default behavior (even in dev mode) because it'll have a big performance impact. We'd be computing styles for every Layout component on every render.
